Modules/UpendoDnnSimpleAuthProvider/App_LocalResources/Login.ascx.resx

```diff
@@ -156,4 +156,7 @@
   <data name="LoginHeader.Text" xml:space="preserve">
     <value>Please enter your username to send your verification code.</value>
   </data>
+  <data name="AccessDenied" xml:space="preserve">
```

Comment:
@ArielBlanco1990 I haven't tested anything yet, but I'm guessing this won't work.  It's missing the `.Text` suffix.

Reply:
I added it directly from VisualStudio. But you're right, I don't know the reason why the suffix wasn't added automatically.

Reply:
@ArielBlanco1990 It isn't automatic in the localization file.  It's appended by the DNN API though, when calling for.  So the API accepts both `AccessDenied` and `AccessDenied.Text`, but `AccessDenied.Text` must be the key in the localization file, regardless.